internal/users/localentries/getgrent_test.go

```diff
@@ -9,24 +11,47 @@ import (
 func TestGetGroupEntries(t *testing.T) {
 	t.Parallel()
 
-	got, err := GetGroupEntries()
-	require.NoError(t, err, "GetGroupEntries should never return an error")
-	require.NotEmpty(t, got, "GetGroupEntries should never return an empty list")
+	// Ensure the requests can be parallelized
+	for idx := range 10 {
+		t.Run(fmt.Sprintf("iteration_%d", idx), func(t *testing.T) {
+			t.Parallel()
+
+			got, err := GetGroupEntries()
```

Comment:
The first thing `GetGroupEntries` does is to take a lock. Is there any doubt that this works, which justifies that we run this test 10 times?

Reply:
Well... Given how fast it it is, I think is fine, and at least in case we rewrite it in future, we can ensure that it still works in parallel as expected

Reply:
Ok, `go test ./internal/users/localentries/...` still runs in 0.2 seconds on my system, so I'm fine with it.